### DISCUSSION TITLE
`get_lldp_neighbors_detail.ptl` doesn't exist on `nxos_ssh`

### DISCUSSION THREAD

**Discussion by remchuk** [napalm-automation/napalm#2014] [Q&A]
When I try to get_lldp_neighbors() from a switch with an nxos_ssh, I get an answers and an error, why ?

I get this error :
```
Errorcode="textfsmExtractorErr01" in naplam.base.helpers with systemMessage="[Errno2] No such file or directory: '/use/local/lib/python3.11/site-packages/naplam/nxos_ssh/utils/textfsm_templates/show_lldp_neighbors_detail.tpl'"
```

Is there a way to remove this error? Or is there an alternative for this?

**Comment by ktbyers** [comment]
@remchuk Which version of NAPALM is this from?

Also you need to post the relevant section of your code and the full Python exception stack trace (so we can see what is causing the error).

**Reply by remchuk** [reply]
> @remchuk Which version of NAPALM is this from?
> 
> Also you need to post the relevant section of your code and the full Python exception stack trace (so we can see what is causing the error).

I will check and answer , i totally forgot

**Reply by remchuk** [reply]
> @remchuk Which version of NAPALM is this from?
> 
> Also you need to post the relevant section of your code and the full Python exception stack trace (so we can see what is causing the error).

Its looks. Like the file show_lldp_neighbors_detail.tpl is missing inside the nxos_ssh , but exists in the nxos.

Napalm 4.0.0
